DEVELOPER_GUIDE.md

```diff
@@ -22,6 +22,8 @@ So you want to contribute code to this project? Excellent! We're glad you're her
     - `cdk deploy OpenSearchMetricsNginxReadonly`: To deploy the dashboard read only setup.
     - `cdk deploy OpenSearchWAF`: To deploy the AWS WAF for the project ALB's.
     - `cdk deploy OpenSearchMetrics-Monitoring`: To deploy the alerting stack which will monitor the step functions and URL of the project coming from [METRICS_HOSTED_ZONE](https://github.com/opensearch-project/opensearch-metrics/blob/main/infrastructure/lib/enums/project.ts)
+    - `cdk deploy OpenSearchMetrics-GitHubAutomationApp-Secret`: Creates the GitHub app secret which will be used during the GitHub app runtime.
```

Comment:
I thought we already have the secrets created?

Reply:
Ya this is created Peter but adding it to the DEVELOPER_GUIDE.md, once we call the `OpenSearchMetrics-GitHubAutomationApp` stack the `OpenSearchMetrics-GitHubAutomationApp-Secret` is created as dependency. Also we can call the `OpenSearchMetrics-GitHubAutomationApp-Secret` separately hence recording this to the DEVELOPER_GUIDE.md.